_posts/2019-11-28-DevOpsDays-2019.md

```diff
@@ -0,0 +1,259 @@
+
+And by the time he became a guru, it was already obsolete, and it was no longer the thing you did. 
+You do Microsoft or something else that replaces Novell. 
+The tech landscape changes every couple of years and thus you have to learn things again sometimes. 
+But that doesn't really help when the Dunning-Kruger effect applies when learning new stuff. 
+Most people don't really discuss that certain things are new to them and that they are not really good at it yet. 
+And they feel insecure sometimes. 
+
+<img alt="Dunning-Kruger Effect" src="{{ '/img/2019-11-28-devopsdays-2019/dunning-kruger-effect.png' | prepend: site.baseurl }}" class="image fit" style="margin:0px auto; max-width: 750px;">
+
+There are certain things you can do to reduce the feeling of the Imposter Syndrome (or to recalibrate your 'internal ruler' so to speak).
+
+* **Nothing compares to you.** Every time you say something negative about yourself, say two things that are positive about yourself. 
+It is as easy as that to retrain your internal NLP filter. 
+At some point, your negative stuff will diminish, and your positive stuff will stay. 
+Recommended to try this in front of a mirror.
+
+* **"I will not compare myself to strangers on the internet."** 
+You shouldn't compare yourself to others, especially to others on the internet, because they are the worst.
+
+* **Practice on giving compliments to others. And learn to receive them.** 
+They can come from different perspectives. 
+For a kid that just learned to tie shoelaces for example is an accomplishment because it is new to him/her. 
+Although it means "nothing" to you because you've learned that a long time ago.
+
+* **Learn how others make stuff and let others learn from you.** 
+Other people no matter how professional they are, make mistakes as well and can learn things from you and you can learn from them.
+    * Pair programming
+    * Pair review
+    * Celebrate your failure
+    * Speak publicly
+
+* **Find something your passionate about and keep tremendously interested in it.**
+    * A hobby where you can screw things up without consequences
+
+* .. etc
+
+## Fight, Flight or Freeze – Releasing Organizational Trauma, by Matty Stratton
+Imagine a Zebra grazing in the savanna of the African wilderness. 
+Then suddenly a predator goes after the zebra. 
+Of course, the zebra will run for his life (the flee state). 
+When the zebra is caught by the predator, it 'freezes' because the zebra’s nervous system is overwhelmed, and it has no further solutions anymore. 
+In the hope the predator will drop him and goes on. 
+After the zebra has managed to get away it literally shakes all his stress on and goes back to rest like nothing happened. 
+Humans can react the same way zebras do, although we are not a zebra. 
+Humans on the other hand can experience traumas long after a dreadful situation has occurred in the way on our human brain works.
```

Comment:
```suggestion
Humans on the other hand can experience traumas long after a dreadful situation has occurred because of the way our human brain works.
```